### PR TITLE
update schema and data sql

### PR DESCRIPTION
**Pull request by tomasesquivelgc**
# In this branch: 
  - I update the table by adding a new species column.
  - I add more entries to the table.
  - I add required queries.

## Transactions screenshots: 
image
image
image
image
image
image

## Queries screenshots:
image
image
image
image
image
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,6 +6,7 @@ CREATE TABLE public.vet_clinic
     escape_attempts integer NOT NULL,
     neutered boolean NOT NULL,
     weight_kg numeric NOT NULL,
+    species text,
```
Comment (descholar-ceo) [inline schema.sql]: - [x] Editing already ran migration is not a recommended way to add new fields actually, you should use this statement `ALTER TABLE animals ADD species varchar(255);` instead!

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,67 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
```
Comment (descholar-ceo) [inline queries.sql:22]: - [x] This queries are supposed to be ran inside a transaction

File: queries.sql [hunk 3]
```diff
@@ -5,4 +5,67 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+
+SELECT
+    species,
+    AVG(escape_attempts) AS avg_escape_attempts
+FROM
+    animals
+WHERE
+    date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
+GROUP BY
+    species;
```
Comment (descholar-ceo) [inline queries.sql:85]: - [x] We are missing a query from this file which sets a animal's specie as null

```sql
UPDATE animals
   SET species = 'unspecified'; 
```

File: queries.sql [hunk 4]
```diff
@@ -5,4 +5,67 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+SAVEPOINT weight_update_savepoint;
+
+-- Update all animals' weight to be their weight multiplied by -1
+UPDATE animals SET weight_kg = -weight_kg;
+
+-- Rollback to the savepoint
+ROLLBACK TO SAVEPOINT weight_update_savepoint;
+
+-- Update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals SET weight_kg = -weight_kg WHERE weight_kg < 0;
```
Comment (descholar-ceo) [inline queries.sql]: - [x] [OPTIONAL] Just for the sake of clean and readable code, I would recommend you to multiply `weight_kg` with -1 in order to get a negative value

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @tomasesquivelgc 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 Your PR is descriptive with the screenshots you added that make its context clear ✔️ 
👍🏽 Almost all requirements are met except a few points of improvements ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @tomasesquivelgc  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy


## Highlights
- Correct data inserted into the table✔✔
- Queries are written correctly :heavy_check_mark: 
- Screenshots are correct and added to pull description :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**